README.md

```diff
@@ -153,6 +153,15 @@ For Puppet 4.2.2+ (and 3.8.2) we've lessened the possibility of the issue occurr
 
 If you are affected by this issue, a known workaround is to download the `.tar.gz` from the forge and use `puppet module install` using the file rather than directly from the forge.
 
+## Running Puppet and DSC without Administrative Privleges
+
+Currently you cannot invoke DSC Resources using a user account without administrative privleges. The underlying CIM implementation DSC uses for DSC Resource invocation requires adminstrative credentials to function.
```

Comment:
~~Currently you cannot invoke DSC Resources using a user account without administrative privleges.~~ While there are avenues for using Puppet as a non-administrative account, DSC is limited to only accounts with administrative privileges.